src/classes/frDonorTest.cls

```diff
@@ -40,11 +40,7 @@
 @isTest
 public class frDonorTest {
     static testMethod void fromContact_test() {
-        Contact testContact = new Contact(
```

Comment:
Somewhat unrelated changes, I think these were intended to be in a different PR but did not get included when pulling from SF into my local working copy